Common/src/Auth/Authentication/Authenticator.cs

```diff
@@ -177,54 +185,70 @@ protected override async Task<AuthenticateResult> HandleAuthenticateAsync()
   }
 
   public async Task<UserIdentity?> GetIdentityFromCertificateAsync(string cn,
-                                                              string fingerprint)
+                                                                   string fingerprint)
   {
     logger_.LogDebug("Authenticating request with CN {CN} and fingerprint {Fingerprint}",
                      cn,
                      fingerprint);
     var result = await authTable_.GetIdentityAsync(cn,
-                                                   fingerprint,
-                                                   new CancellationToken(false))
+                                                   fingerprint)
```

Comment:
Where is the cancellation token? I am pretty sure we want to keep it in signature of authTable.

Reply:
It's still there, but now it's an optional with default value "default"